fabric-fluids-v1/src/main/java/net/fabricmc/fabric/api/fluids/containers/SimpleFluidContainer.java

```diff
@@ -0,0 +1,113 @@
+package net.fabricmc.fabric.api.fluids.containers;
+
+import java.util.Arrays;
+import java.util.Iterator;
+
+import com.google.common.collect.Iterables;
+
+import net.minecraft.util.Identifier;
+
+import net.fabricmc.fabric.api.fluids.containers.volume.FluidVolume;
+import net.fabricmc.fabric.api.fluids.containers.volume.SimpleFluidVolume;
+
+public class SimpleFluidContainer implements FluidContainer {
+	private final Iterable<FluidContainer> containers;
+
+	public SimpleFluidContainer(FluidContainer... containers) {
+		this(Arrays.asList(containers));
+	}
+
+	public SimpleFluidContainer(Iterable<FluidContainer> containers) {
+		this.containers = containers;
+	}
+
+	@Override
+	public FluidVolume drain(Identifier fluid, long amount, boolean simulate) {
```

Comment:
Why is simulation ignored in this method?

Reply:
because it doesn't have an internal state that gets mutated

Reply:
the FluidContainers within this one have the real implementation

Reply:
But you don't pass the simulation state to the contained containers, you always pass `false`?